source/api/commands/exec.md

```diff
@@ -89,12 +89,11 @@ cy.exec('npm run my-script').its('stdout').should('contain', 'Done running the s
 ### Write to a file to create a fixture from response body
 
 ```javascript
-cy.server()
-cy.route('POST', '/comments').as('postComment')
+cy.http('POST', '/comments').as('postComment')
 cy.get('.add-comment').click()
-cy.wait('@postComment').then((xhr) => {
-  cy.exec(`echo ${JSON.stringify(xhr.responseBody)} >cypress/fixtures/comment.json`)
-  cy.fixture('comment.json').should('deep.eq', xhr.responseBody)
+cy.wait('@postComment').then(({ response }) => {
+  cy.exec(`echo ${JSON.stringify(response.body)} >cypress/fixtures/comment.json`)
+  cy.fixture('comment.json').should('deep.eq', response.body)
```

Comment:
This example doesn't work, so is not the equivalent of the previous version. The assertion seems to fail now because there are newlines in the body. (I'm still reviewing but this is one example of a change needed that I've run into)

```js
  it('cy.route() - passes', () => {
    cy.visit('https://example.cypress.io/commands/network-requests')
    cy.server()
    cy.route('POST', '/comments').as('postComment')
    cy.get('.network-post').click()
    cy.wait('@postComment').then((xhr) => {
      cy.exec(`echo ${JSON.stringify(xhr.responseBody)} >cypress/fixtures/comment1.json`)
      cy.fixture('comment.json').should('deep.eq', xhr.responseBody)
    })
  })

  it('cy.http() - fails', () => {
    cy.visit('https://example.cypress.io/commands/network-requests')
    cy.http('POST', '**/comments').as('postComment')
    cy.get('.network-post').click()
    cy.wait('@postComment').then(({ response }) => {
      cy.exec(`echo ${JSON.stringify(response.body)} >cypress/fixtures/comment2.json`)
      cy.fixture('comment.json').should('deep.eq', response.body)
    })
  })
```

<img width="823" alt="Screen Shot 2020-11-19 at 8 17 43 PM" src="https://user-images.githubusercontent.com/1271364/99674512-88058b80-2aa4-11eb-9400-439f73f2aaa5.png">